buildmodel/commands.go

```diff
@@ -136,8 +134,13 @@ func BindPRFlags() *PRFlags {
 		origin: flag.String("origin", "git@github.com:microsoft/go-images", "Submit PR to this repo. \n[Need fetch Git permission.]"),
 		to:     flag.String("to", "", "Push PR branch to this Git repository. Defaults to the same repo as 'origin' if not set.\n[Need push Git permission.]"),
 
-		githubPAT:         flag.String("github-pat", "", "Submit the PR with this GitHub PAT, if specified."),
-		githubPATReviewer: flag.String("github-pat-reviewer", "", "Approve the PR and turn on auto-merge with this PAT, if specified. Required, if github-pat specified."),
+		GHAuthFlags: GHAuthFlags{
```

Comment:
This should be a function similar to how `BindUpdateFlags` and `sync.BindAzDOVariableFlags` work.